linux-package.sh

```diff
@@ -16,15 +16,14 @@ javapackager -deploy \
     -BappVersion=$ver \
     -Bcategory=Office,Finance \
     -BlicenseType=Apache \
+    -BlicenseFile=LICENSE \
     -Bemail=contact@vinumeris.com \
     -Bicon=client/icons/icon.png \
     -native deb \
     -name lighthouse \
-    -title Lighthouse \
+    -title "Lighthouse: a peer to peer crowdfunding app that uses Bitcoin" \
     -vendor Vinumeris \
     -outdir deploy \
     -appclass lighthouse.Main \
-    -srcfiles $build \
+    -srcdir updates/builds/processed \
```

Comment:
Is this change really correct? This looks like it might try and package every single JAR in that directory instead of only the one that corresponds to the right version number.